taxi/urls.py

```diff
@@ -46,6 +49,22 @@
     path(
         "drivers/<int:pk>/", DriverDetailView.as_view(), name="driver-detail"
     ),
+    path("drivers/create/", DriverCreateView.as_view(), name="driver-create"),
+    path(
+        "drivers/<int:pk>/update/",
+        DriverUpdateView.as_view(),
+        name="driver-update"
+    ),
+    path(
+        "drivers/<int:pk>/delete/",
+        DriverDeleteView.as_view(),
+        name="driver-delete"
+    ),
+    path(
+        "cars/<int:pk>/assign_delete/",
+        CarDetailView.as_view(),
+        name="car-assign-delete"
+    ),
```

Comment:
there is no need to create a new link here

Reply:
Why not? May you explain please?